### PR TITLE
misc natives

### PR DESCRIPTION
**Pull request by slavexe**

### CODE REVIEW

File: natives.json [hunk 1]
```diff
@@ -94694,33 +94694,33 @@
 			"build": "323"
 		},
 		"0x45C597097DD7CB81": {
-			"name": "ADD_COVER_BLOCKING_AREA",
+			"name": "ADD_COVER_BLOCKING_IN_ANGLED_AREA",
```
Comment (alloc8or) [inline natives.json]: Why did you change a confirmed name?

Comment (slavexe) [reply natives.json]: I didn't notice it was confirmed. I reverted the name changes and I double checked the parameters are still correct. These natives must be referring to normal areas, not angled areas as it seemed based on plotting positions the scripts were using in Codewalker.

Comment (slavexe) [reply natives.json]: Also, I just realized these obviously cannot be angled areas because angled areas require an extra width float, which neither of these functions have.

Comment (alloc8or) [reply natives.json]: > I didn't notice it was confirmed.

Anything not starting with an underscore is confirmed and should never be changed, except in the rare case of a hash collision.

Comment (slavexe) [reply natives.json]: Yeah I read Alexander Blade's post on GTA Forums regarding native naming a while ago but forgot about that part. Good to see so many names have been confirmed already.
